tests/test_urls.php

```diff
@@ -230,8 +230,18 @@ function test_gp_url_public_root_has_no_trailing_slash_when_permalinks_have_no_t
 	 * @ticket gh-203
 	 */
 	function test_gp_url_public_root_has_a_trailing_slash_when_permalinks_have_a_trailing_slash() {
+		GLOBAL $wp_tests_options, $wp_rewrite;
+		
 		$this->set_permalink_structure( '/%postname%/' );
+		$wp_rewrite->use_trailing_slashes = true;
+		update_option( 'permalink_structure', '/%postname%/' );
```

Comment:
That's interesting, shouldn't those two lines be handled by `WP_UnitTestCase->set_permalink_structure()`?